meta/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 galaxy_info:
+  role_name: storage
+  namespace: linux_system_roles
```

Comment:
```suggestion
```

Reply:
we can't use this - I know that ansible-lint complains about it, but ansible-lint doesn't work well with the legacy role format, it prefers collection format - this is why tox-lsr has a workaround for this issue - https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/test_scripts/ansible-lint-helper.sh - and why our github action ci ansible-lint task has the same workaround - https://github.com/linux-system-roles/storage/blob/main/.github/workflows/ansible-lint.yml#L29
so if you don't want to use `tox-lsr` to run ansible-lint, then please either ignore these errors, or edit meta/main.yml locally but do not commit it

Reply:
My bad, I didn't plan to include this in the commit -- I just made the change (based on the GH action workflow) to be able to run ansible lint locally and then used `git add -u` without thinking. Removed.